Gordon360/Controllers/MembershipsController.cs

```diff
@@ -228,12 +228,12 @@ public async Task<ActionResult<MembershipView>> SetGroupAdminAsync(int membershi
             return Ok(result);
         }
 
-        /// <summary>Update an existing membership item to be private or not</summary>
+        /// <summary> Update an existing membership item to be private or not </summary>
```

Comment:
```suggestion
        /// <summary>Update an existing membership item to be private or not</summary>
```
The other endpoints don't have spaces here.